src/actionCreators/resources.js

```diff
@@ -22,10 +22,10 @@ import ResourceStateBuilder from 'ResourceStateBuilder'
 import _ from 'lodash'
 
 // A thunk that updates (saves) an existing resource in Trellis
-export const update = (resourceKey, currentUser, errorKey) => (dispatch, getState) => {
+export const update = (resourceKey, currentUser, errorKey) => async (dispatch, getState) => {
```

Comment:
This may or may not work, and may have side effects because I didn't bother to chase the promises up any farther in the stack.